Bottom: 24b1f72a2188e4ef5d16316519a044d9d51b9958
Top:    5e59b527161f166ad30ab5343076c3b1385a9349
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 22:26:00 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index e43ec87..17cd81d 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -1057,9 +1057,9 @@ ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
   glLoadIdentity ();
 
   glTranslatef (widget->allocation.width / 2., widget->allocation.height / 2., 0);
-  glMultMatrixf (view_matrix);
+  glMultMatrixf ((GLfloat *)view_matrix);
   glTranslatef (-widget->allocation.width / 2., -widget->allocation.height / 2., 0);
-  glGetFloatv (GL_MODELVIEW_MATRIX, last_modelview_matrix);
+  glGetFloatv (GL_MODELVIEW_MATRIX, (GLfloat *)last_modelview_matrix);
 
   glEnable (GL_STENCIL_TEST);
   glClearColor (gport->offlimits_color.red / 65535.,
